risk/resilient-price-oracle.md

```diff
@@ -135,6 +135,8 @@ The current list of correlated token oracles in Venus is:
 | Core | USDT | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
 | Core | WBTC | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
 | Core | WETH | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
+| Core | DAI | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
+| Core | TUSD | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
```

Comment:
Let's follow alphabetical order inside the Core markets

Reply:
https://github.com/VenusProtocol/venus-protocol-documentation/pull/197/commits/c93a1d8a9c16d3de466310990168250abbefd14e